### PR TITLE
Braces example for multiline conditionals

### LINKED ISSUE #283: Clarify position of open brace in multiline conditionals
From https://mail.python.org/pipermail/python-dev/2017-June/148223.html

> But the latter example continuation lines are intended at the same level as the following block of code. I propose to make exception for that case and allow moving an open brace to the start of the next line.

I agree with Serhiy; pull request coming momentarily.

### PR DESCRIPTION
**Pull request by warsaw**
Closes #283

### CODE REVIEW

File: pep-0007.txt [hunk 1]
```diff
@@ -124,11 +124,13 @@ Code lay-out
                    type->tp_name);
 
 * When you break a long expression at a binary operator, the
-  operator goes at the end of the previous line, e.g.::
+  operator goes at the end of the previous line, and braces should be
```
Comment (serhiy-storchaka) [inline pep-0007.txt]: Actually I think the brace should be moved only if the indentation of continuation lines matches the indentation of the following code. In the following case the brace should be kept at the end of the line: 
```
        if (PyErr_WarnFormat(PyExc_DeprecationWarning, 1,
                             "invalid escape sequence '\\%c'",
                             *first_invalid_escape) < 0) {
            Py_DECREF(result);
            return NULL;
        }
```

Comment (warsaw) [inline pep-0007.txt]: That actually looks a bit weird to me, though I can understand the rationale.  I'd personally prefer a rule that states the open brace not to be hanging in all multiline conditional cases.

File: pep-0007.txt [hunk 2]
```diff
@@ -124,11 +124,13 @@ Code lay-out
...
+  formatted as show.  E.g.::
```
Comment (csabella) [inline pep-0007.txt]: Should this be 'as shown'?

Comment (warsaw) [inline pep-0007.txt]: Yep!

**Pull request merged**